app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -20,7 +20,8 @@ def ems_inv_to_hashes
       log_header = "MIQ(#{self.class.name}.#{__method__}) Collecting data for EMS name: [#{@ems.name}] id: [#{@ems.id}]"
       $scvmm_log.info("#{log_header}...")
 
-      @inventory = ManageIQ::Providers::Microsoft::InfraManager.execute_powershell_json(@connection, INVENTORY_SCRIPT)
+      @script = IO.read(INVENTORY_SCRIPT)
```

Comment:
Does this have to be an instance var?  Looks like it's just used on the next line.